x-pack/libbeat/management/enroll.go

```diff
@@ -22,23 +21,28 @@ const accessTokenKey = "management.accesstoken"
 
 // Enroll this beat to the given kibana
 // This will use Central Management API to enroll and retrieve an access key for config retrieval
-func Enroll(beat *instance.Beat, kibanaConfig *kibana.ClientConfig, enrollmentToken string, force bool) (bool, error) {
+func Enroll(
+	beat *instance.Beat,
+	kibanaConfig *kibana.ClientConfig,
+	enrollmentToken string,
+	force bool,
```

Comment:
The param `force` is not needed anymore, as `configOverwrite` is not used.

Reply:
Excellent catch!